invenio_db/core.py

```diff
@@ -59,6 +61,9 @@ def init_db(self, app, entrypoint_name='invenio_db.models', **kwargs):
             'sqlite:///' + os.path.join(app.instance_path, app.name + '.db')
         )
         app.config.setdefault('SQLALCHEMY_ECHO', app.debug)
+        app.config.setdefault('DB_VERSIONING_USER_MODEL', None)
```

Comment:
After musings with @lnielsen the prefered way is still this one if the `DB_VERSIONING_USER_MODEL` value is not set:
```python
try:
    pkg_resources.get_distribution('invenio_accounts')
except pkg_resources.DistributionNotFound:
    user_cls = None
else:
    from invenio_accounts.models import User as user_cls
```